pyanaconda/ui/gui/spokes/datetime_spoke.py

```diff
@@ -254,7 +254,8 @@ def run(self):
                     new_servers.append(row[0])
 
             if flags.can_touch_runtime_system("save NTP servers configuration"):
-                ntp.save_servers_to_config(new_servers)
+                ntp.save_servers_to_config(new_servers, conf_file_path=ntp.CHRONY_CONFIG_FILE)
+                ntp.save_servers_to_config(new_servers, conf_file_path=ntp.NTP_CONFIG_FILE)
```

Comment:
We only need the chrony config here as this is for the installation environment to use the configured servers and the installation environment uses chrony.